udata/core/discussions/csv.py

```diff
@@ -0,0 +1,22 @@
+from udata.frontend import csv
+
+from .models import Discussion
+
+
+@csv.adapter(Discussion)
+class DiscussionCsvAdapter(csv.Adapter):
+    fields = (
+        "id",
+        "user",
+        "subject",
+        ("subject_class", "subject._class_name"),
+        ("subject_id", "subject.id"),
+        "title",
+        ("size", lambda o: len(o.discussion)),
+        ("participants", lambda o: ",".join(set(str(msg.posted_by.id) for msg in o.discussion))),
+        ("messages", lambda o: "\n".join(msg.content.replace("\n", " ") for msg in o.discussion)),
+        "created",
+        "closed",
+        ("closed_by_fullname", "closed_by"),

```

Comment:
Woops, coming after the battle but `closed_by_fullname` breaks compatibility. I think we can keep `closed_by` and `closed_by_id`, similarly to what is used in other csv adapters, ex: `organization` (name) and `organization_id`.

Reply:
https://github.com/opendatateam/udata/pull/3153